releasenotes/notes/pending-deprecate-qnns-ef003277a841096d.yaml

```diff
@@ -0,0 +1,12 @@
+---
+deprecations:
+  - |
+    Class :class:`qiskit_machine_learning.neural_networks.CircuitQNN` is pending deprecation and
+    is superseded by :class:`qiskit_machine_learning.neural_networks.SamplerQNN`.
+
+    Class :class:`qiskit_machine_learning.neural_networks.OpflowQNN` is pending deprecation and
+    is superseded by :class:`qiskit_machine_learning.neural_networks.EstimatorQNN`.
+
+    Class :class:`qiskit_machine_learning.neural_networks.TwoLayerQNN` is pending deprecation and
+    has no direct replacement. Please make use of
+    :class:`qiskit_machine_learning.neural_networks.EstimatorQNN` instead.
```

Comment:
Should we note all of the above will be deprecated and subsequently removed in future releases. By default you don't get to see pending deprecation messages so while that extra info is in them maybe a single sentence covering all of the above classes would be helpful here too.

Reply:
Added "These classes will be deprecated in a future release and subsequently removed after that."
Initially the reno was copied from the QK deprecation reno. I updated it similarly as well.